src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -186,7 +190,7 @@ extension ExposureDetectionViewController {
 		tableView.reloadData()
 	}
 
-	private func updateCheckButton() {
+	private func updateCheckButton(_ time: String? = nil) {
```

Comment:
A comment here, just shortly explaining what this method does (in case of side effects) might help

Reply:
great! thanks for adding the comment.